release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -828,6 +828,11 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+|Simplified disconnected mirroring with `oc`
```

Comment:
Not sure how this is handled. Can this specify `Simplified disconnected mirroring with the oc plugin `oc-mirror`?